clang/test/CodeGen/RISCV/attr-hw-shadow-stack.c

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang_cc1 -triple riscv64-linux-unknown -target-feature +zimop -emit-llvm -o - %s -fcf-protection=return | FileCheck -check-prefix=NOTIGNORELISTED %s
+// RUN: %clang_cc1 -triple riscv64-linux-unknown -target-feature +zimop -emit-llvm -o - %s | FileCheck -check-prefix=IGNORELISTED %s
+
+int foo(int *a) { return *a; }
+
+// CHECK: define i32 @foo(ptr %a)
```

Comment:
It looks like this line is not checked if you specify the `--check-prefix` option to FileCheck:

> --check-prefix prefix
> FileCheck searches the contents of match-filename for patterns to match. By default, these patterns are prefixed with “CHECK:”. If you’d like to use a different prefix (e.g. because the same input file is checking multiple different tool or options), the [--check-prefix](https://llvm.org/docs/CommandGuide/FileCheck.html#cmdoption-FileCheck-check-prefix) argument allows you to specify (without the trailing “:”) one or more prefixes to match. Multiple prefixes are useful for tests which might change for different run options, but most lines remain the same.

(ref. https://llvm.org/docs/CommandGuide/FileCheck.html)

Reply:
It's surely not used so I've removed it
(This test is modified from `clang/test/CodeGen/shadowcallstack-attr.c`, so as the line, but seems like it isn't used in the original file either)